ivy/functional/frontends/tensorflow/activations.py

```diff
@@ -16,6 +16,10 @@ def linear(x):
     return x
 
 
+def relu(x):

```

Comment:
you should also add `name=None` as an argument because `tf.nn.relu` has it

Reply:
ok